static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-mobile-design-test.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import closeCentralIcon from 'svgs/icon/close-central.svg';
+import arrowWhiteRight from 'svgs/icon/arrow-white-right.svg';
+import applyPayMark from 'svgs/acquisitions/apple-pay-mark.svg';
+import config from 'lib/config';
+import { applePayApiAvailable } from 'lib/detect';
+import { acquisitionsBannerTickerTemplate } from 'common/modules/commercial/templates/acquisitions-banner-ticker';
+
+export const acquisitionsBannerMobileDesignTestTemplate = (
+    params: EngagementBannerTemplateParams
+): string => {
+    const applePayLogo = applePayApiAvailable ? applyPayMark.markup : '';
+    return `
+        <div class="engagement-banner__close">
+            <button class="button engagement-banner__close-button js-site-message-close js-engagement-banner-close-button" data-link-name="hide release message">
+                <span class="u-h">Close</span>
+                ${closeCentralIcon.markup}
+            </button>
+        </div>
+        <div class="engagement-banner__container">
+            <div class="engagement-banner__text engagement-banner__text--mobile">
```

Comment:
doesn't look like `engagement-banner__text--mobile` and `engagement-banner__header--mobile` have any styles associated with them?

Reply:
👍 leftover from when i was trying to do both variants on one template